src/lib/Zikula/Bundle/CoreBundle/Twig/Extension/CoreExtension.php

```diff
@@ -192,12 +211,98 @@ public function arrayUnset($array, $key)
     }
 
     /**
-     * Returns the name of the extension.
+     * @param $code
+     * @return string
+     */
+    public function languageName($code)
+    {
+        return \ZLanguage::getLanguageName($code);
+    }
+
+    /**
+     * @param $string
+     * @return string
+     */
+    public function safeHtml($string)
+    {
+        return \DataUtil::formatForDisplayHTML($string);
+    }
+
+    /**
+     * @todo this maybe should be rewritten to a Tag (Token Parer and Node Visitor) and called {% pageSetVar name = "value" %}
      *
-     * @return string The extension name
+     * @param null $name
+     * @param null $value
      */
-    public function getName()
+    public function pageSetVar($name = null, $value = null)
     {
-        return 'zikulacore';
+        if (empty($name) || empty($value)) {
+            throw new \InvalidArgumentException(__('Empty argument at') . ':' . __FILE__ . '::' . __LINE__);
+        }
+
+        if (in_array($name, array('stylesheet', 'javascript'))) {
+            $value = explode(',', $value);
```

Comment:
I vote for removing this. There is no harm in calling pageSetVar multiple times for multiple js / css files.